src/OpenOfficeConvert/OOStyles.cs

```diff
@@ -2851,6 +2854,31 @@ private void WritePageLayoutStylePropertySeven()
             /* pm7 ends*/
         }
 
+	    private void WritePageLayoutStylePropertyEight()
+	    {
+		    /* pm8 starts - Non Footer settings */
+		    _writer.WriteStartElement("style:page-layout");
+		    _writer.WriteAttributeString("style:name", "pm8");

```

Comment:
It would be good to use the constant on this line too.

Reply:
Ok